yt_dlp/extractor/cspan.py

```diff
@@ -126,8 +130,11 @@ def add_referer(formats):
                         ext = 'vtt'
                     subtitle['ext'] = ext
             ld_info = self._search_json_ld(webpage, video_id, default={})
-            title = get_element_by_class('video-page-title', webpage) or \
-                self._og_search_title(webpage)
+            try:
+                title = get_element_by_class('video-page-title', webpage) or \
+                    self._og_search_title(webpage)
+            except Exception:
+                title = 'No title'
```

Comment:
Why?

Reply:
Some C-Span website pages have an incorrect closing H tag, like this:
`<h3 class='video-page-title' itemprop='name'>U.S. Senate</h2>`

Because of this, the file can't be downloaded with this error message:
`ERROR: matching opening tag for closing h2 tag not found`

It's a workaround for such issues.

Reply:
example?

Reply:
```suggestion
            title = get_element_by_class('video-page-title', webpage) or self._og_search_title(webpage)
```

Merge the PR with master and this will work

Reply:
> Merge the PR with master and this will work

I merged, but it still did not work. The same error:
`yt_dlp.compat.compat_HTMLParseError: matching opening tag for closing h2 tag not found`

Reply:
right, my bad. But catch only the relevant exception